test/GSN/GSNBouncerSignature.test.js

```diff
@@ -17,58 +19,107 @@ contract('GSNBouncerSignature', function ([_, signer, other]) {
     });
 
     it('rejects unsigned relay requests', async function () {
-      await gsn.expectError(this.recipient.mockFunction({ value: 0, useGSN: true }));
+      await gsn.expectError(
+        this.recipient.mockFunction({ value: 0, useGSN: true })
+      );
     });
 
     it('rejects relay requests where some parameters are signed', async function () {
-      const approveFunction = async (data) =>
+      const approveFunction = async data =>
         fixSignature(
           await web3.eth.sign(
             web3.utils.soliditySha3(
               // the nonce is not signed
               // eslint-disable-next-line max-len
-              data.relayerAddress, data.from, data.encodedFunctionCall, toBN(data.txFee), toBN(data.gasPrice), toBN(data.gas)
-            ), signer
+              data.relayerAddress,
+              data.from,
+              data.encodedFunctionCall,
+              toBN(data.txFee),
+              toBN(data.gasPrice),
+              toBN(data.gas)
+            ),
+            signer
           )
         );
 
-      await gsn.expectError(this.recipient.mockFunction({ value: 0, useGSN: true, approveFunction }));
+      await gsn.expectError(
+        this.recipient.mockFunction({ value: 0, useGSN: true, approveFunction })
+      );
     });
 
     it('accepts relay requests where all parameters are signed', async function () {
-      const approveFunction = async (data) =>
+      const approveFunction = async data =>
         fixSignature(
           await web3.eth.sign(
             web3.utils.soliditySha3(
               // eslint-disable-next-line max-len
-              data.relayerAddress, data.from, data.encodedFunctionCall, toBN(data.txFee), toBN(data.gasPrice), toBN(data.gas), toBN(data.nonce), data.relayHubAddress, data.to
-            ), signer
+              data.relayerAddress,
+              data.from,
+              data.encodedFunctionCall,
+              toBN(data.txFee),
+              toBN(data.gasPrice),
+              toBN(data.gas),
+              toBN(data.nonce),
+              data.relayHubAddress,
+              data.to
+            ),
+            signer
           )
         );
 
-      const { tx } = await this.recipient.mockFunction({ value: 0, useGSN: true, approveFunction });
+      const { tx } = await this.recipient.mockFunction({
+        value: 0,
+        useGSN: true,
+        approveFunction,
+      });
 
-      await expectEvent.inTransaction(tx, GSNBouncerSignatureMock, 'MockFunctionCalled');
+      await expectEvent.inTransaction(
+        tx,
+        GSNBouncerSignatureMock,
+        'MockFunctionCalled'
+      );
     });
 
     it('rejects relay requests where all parameters are signed by an invalid signer', async function () {
-      const approveFunction = async (data) =>
+      const approveFunction = async data =>
         fixSignature(
           await web3.eth.sign(
             web3.utils.soliditySha3(
               // eslint-disable-next-line max-len
```

Comment:
We should remove these if they are no longer needed.

Reply:
Ah good point.